substrate/test-utils/runtime/src/lib.rs

```diff
@@ -209,19 +234,26 @@ decl_runtime_apis! {
 		fn vec_with_capacity(size: u32) -> Vec<u8>;
 		/// Returns the initialized block number.
 		fn get_block_number() -> u64;
-
 		/// Test that `ed25519` crypto works in the runtime.
 		///
-		/// Returns the signature generated for the message `ed25519` and the public key.
-		fn test_ed25519_crypto() -> (ed25519::AppSignature, ed25519::AppPublic);
+		/// Returns the signature generated for the message `ed25519` both the public key and proof of possession.
+		fn test_ed25519_crypto() -> (ed25519::AppSignature, ed25519::AppPublic, Ed25519Pop);
 		/// Test that `sr25519` crypto works in the runtime.
 		///
-		/// Returns the signature generated for the message `sr25519`.
-		fn test_sr25519_crypto() -> (sr25519::AppSignature, sr25519::AppPublic);
+		/// Returns the signature generated for the message `sr25519` both the public key and proof of possession.
+		fn test_sr25519_crypto() -> (sr25519::AppSignature, sr25519::AppPublic, Sr25519Pop);
 		/// Test that `ecdsa` crypto works in the runtime.
 		///
-		/// Returns the signature generated for the message `ecdsa`.
-		fn test_ecdsa_crypto() -> (ecdsa::AppSignature, ecdsa::AppPublic);
+		/// Returns the signature generated for the message `ecdsa` both the public key and proof of possession.
+		fn test_ecdsa_crypto() -> (ecdsa::AppSignature, ecdsa::AppPublic, EcdsaPop);
+		/// Test that `bls381` crypto works in the runtime
+		///
+		/// Returns both the proof of possession and public key.
+		fn test_bls381_crypto() -> (Bls381Pop, Bls381Public);
+		/// Test that `ecdsa_bls381_crypto` works in the runtime
+		///
+		/// Returns both the proof of possession and public key.
+		fn test_ecdsa_bls381_crypto() -> (EcdsaBls381Pop, EcdsaBls381Public);
```

Comment:
Nit. As the other methods returns <Signature, Public, PoP>, being pedantic I expect to see these new BLS methods to return <Signature, Public, PoP> (yeah in the same order as well :-D )